x-pack/plugins/security_solution/public/timelines/components/timeline/query_bar/index.tsx

```diff
@@ -47,6 +48,42 @@ export interface QueryBarTimelineComponentProps {
   updateReduxTime: DispatchUpdateReduxTime;
 }
 
+const SearchBarContainer = styled.div`
+  /*
+  *
+  * hide search bar default filters as they are distrubing the layout  as shown below
+  *
+  * Filters are displayed with QueryBar so below is how is the layout with default filters.
+  *
+  *
+  *                    --------------------------------
+  *   -----------------|                              |------------
+  *   | DataViewPicker |        QueryBar              | Date      |
+  *   -------------------------------------------------------------
+  *                    |      Filters                 |
+  *                    --------------------------------
+  *
+  * The tree under this component makes sure that default filters are not rendered and we can saperately display
+  * them outside query component so that layout is as below:
+  *
+  *   -----------------------------------------------------------
+  *   | DataViewPicker |      QueryBar              |   Date    |
+  *   -----------------------------------------------------------
+  *   |                       Filters                           |
+  *   -----------------------------------------------------------
+  *
+  * */
+  [data-test-subj='globalQueryBar'] [data-test-subj='filter-items-group'] {
```

Comment:
We should not use `data-test-subj` selectors to add styles, they should be used only for testing and a rename should not imply style changes.

Reply:
@semd , I understand, but some other plugin do not give good classes to be used as selectors. In this case, I enountered that one of the class was changed ( for `filters-item-group`) and all the styles broke apart and that is why I added `data-test-subj` to the global query bar and resorted to that.

For pre-made components, I feel `data-test-subj` specially in this case is much more stable and it is less susceptible to change.

Reply:
Well, if a plugin changes the class of a reusable component and a style breaks it is their fault, it's a regression on their changes, they should have updated the class on all their dependencies, and we should have told them. 

If QA team changes or moves a `data-test-subj` to another element, and a style break, it is our fault for misusing an attribute that is not meant to have styles assigned 🤷. QA should not have to know how to fix CSS.

Just FYI, I am not going to block the PR for this.

Reply:
>Well, if a plugin changes the class of a reusable component and a style breaks it is their fault, it's a regression on their changes, they should have updated the class on all their dependencies, and we should have told them.

the problem is finding these kind of regressions, these do not show up easily in cypress so it is difficult.  

>If QA team changes or moves a data-test-subj to another element, and a style break, it is our fault for misusing an attribute that is not meant to have styles assigned 🤷. QA should not have to know how to fix CSS.

I did not think from this perspective that QA would want to move/touch `data-test-subj` on existing components. 

>Just FYI, I am not going to block the PR for this.
That is okay, this PR is anyway not going in before next week when everyone is back from thanksgiving. I will once more check if there is a better way.

Thanks for you in-depth review of this PR.

Reply:
> I did not think from this perspective that QA would want to move/touch data-test-subj on existing components.

Yeah, not usually. But that's also true for engineers, changing a test subject should no imply doing CSS fixes.

Anyhow, thanks for addressing all the comments.

Reply:
removed `data-test-subj` from styles in ef489cf8aac79a67755710cca186224955c411e8

Reply:
Awesome! Thank you